lib/models/user.go

```diff
@@ -43,7 +43,7 @@ func UserEmailAlreadyExisted(email string) bool {
 func GetUserByEmail(email string) (*User, error) {
 	user := &User{}
 
-	result := database.GetDB().Where("email = ?", email).First(&user)
+	result := database.GetDB().Where("LOWER(email) = LOWER(?)", email).First(&user)
```

Comment:
I saw that you set email to `citext` [here](https://github.com/carryall/go-google-scraper-challenge/blob/develop/database/migrations/20220407163153_create_users.sql#L7).

It doesn't support by omitting the `LOWER` function lor? 🥲

Reply:
Oh I forgot, actually, it do support so I removed this in [618d988](https://github.com/carryall/go-google-scraper-challenge/pull/84/commits/618d988b14d6ca7fabfa9b8ccff16e7c08b3db28)